public/components/datasources/components/manage/associated_objects_tab.tsx

```diff
@@ -0,0 +1,197 @@
+              Last updated at:
+            </EuiText>
+            <EuiText color="subdued" style={{ fontSize: 'small' }}>
+              {lastUpdated}
+            </EuiText>
+          </div>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            data-test-subj="freshButton"
+            iconType="refresh"
+            onClick={fetchAssociatedObjects}
+          >
+            Refresh
+          </EuiButton>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  };
+
+  const noDataMessage = (
+    <EuiEmptyPrompt
+      title={<h2>You have no associated objects</h2>}
+      body={<p>Add or config tables from your data source or use Query Workbench.</p>}
+      actions={
+        <EuiButton
+          color="primary"
+          fill
+          onClick={() => window.open('https://example.com', '_blank')}
+          iconType="popout"
+          iconSide="left"
+        >
+          Query Workbench
+        </EuiButton>
+      }
+    />
+  );
+
+  const columns = [
+    {
+      field: 'name',
+      name: 'Name',
+      sortable: true,
+      'data-test-subj': 'nameCell',
+      render: (name: string) => (
+        <EuiLink href="https://example.com" target="_blank">
+          {name}
+        </EuiLink>
+      ),
+    },
+    {
+      field: 'database',
+      name: 'Database',
+      truncateText: true,
+      render: (database: string) => (
+        <EuiLink href="https://example.com" target="_blank">
+          {database}
+        </EuiLink>
+      ),
+    },
+    {
+      field: 'type',
+      name: 'Type',
+      sortable: true,
+    },
+    {
+      field: 'createdByIntegration',
+      name: 'Created by Integration',
+      sortable: true,
+    },
+    {
+      field: 'accelerations',
+      name: 'Accelerations',
+      sortable: true,
+    },
+    {
+      name: 'Actions',
+      actions: [
+        {
+          name: 'Edit',
+          description: 'Edit this object',
+          type: 'icon',
+          icon: 'discoverApp',
+          onClick: (item: AssociatedObject) => console.log('Edit', item),
+        },
+        {
+          name: 'Delete',
+          description: 'Delete this object',
+          type: 'icon',
+          icon: 'bolt',
+          onClick: (item: AssociatedObject) => console.log('Delete', item),
+        },
+      ],
+    },
+  ];
+
+  const search = {
+    box: {
+      incremental: true,
+      schema: {
+        fields: { name: { type: 'string' }, database: { type: 'string' } },
+      },
+    },
+  };
+
+  const pagination = {
+    initialPageSize: 10,
+    pageSizeOptions: [10, 25, 50],
+  };
+
+  const sorting = {
+    sort: {
+      field: 'name',
+      direction: 'asc',
+    },
+  };
+
+  return (
+    <>
+      <EuiSpacer />
+      <EuiPanel>
+        <AssociatedObjectsHeader />
+        <EuiHorizontalRule />
+        <AccelerationsRecommendationCallout />
+        <EuiSpacer />
+        {associatedObjects.length > 0 ? (
+          <EuiInMemoryTable
```

Comment:
would there be a case that `associatedObjects` gets large and doesn't fit in memory?

Reply:
Good point, I think for the memory fitting issue, we may need to do something like virtualization with react-window to limit the size of the list. However, since this this is a skeleton for now, and lets just leave like this. I will test it when we connect the actual data content, so that we can decide which approaches we need for addressing the memory issue. I will not resolve this comment btw until we have a concrete solution on this.